wasm/types.go

```diff
@@ -30,6 +30,7 @@ const (
 	ValueTypeI64 ValueType = 0x7e
 	ValueTypeF32 ValueType = 0x7d
 	ValueTypeF64 ValueType = 0x7c
+	ValueTypeUnk ValueType = 0x7b
```

Comment:
maybe move this to validation module. this value is an implementation detail for polymorphic typing, and should not exposed outside.

Reply:
ok, agree